description: icf is not supported by ld.bfd, which is used by default in debian
author: Michael Gilbert <mgilbert@debian.org>

--- a/tools/gn/build/gen.py
+++ b/tools/gn/build/gen.py
@@ -299,7 +299,7 @@ def WriteGNNinja(path, platform, host, o
           ldflags.append('-Wl,-strip-all')
 
       # Enable identical code-folding.
-      if not platform.is_darwin():
+      if False:
         ldflags.append('-Wl,--icf=all')
 
     cflags.extend([
